pkgs/development/tools/rq/default.nix

```diff
@@ -11,6 +11,20 @@ rustPlatform.buildRustPackage rec {
     sha256 = "0km9d751jr6c5qy4af6ks7nv3xfn13iqi03wq59a1c73rnf0zinp";
   };
 
+  # This patch is required to prevent build errors with rust edition 2021.
```

Comment:
rust 1.51? 1.50? 1.49?

Reply:
rust edition refers to language spec, which is seperate, but related

Reply:
Patch is removed.